src/reference/v8.json

```diff
@@ -2847,6 +2847,24 @@
           }
         }
       },
+      "semiliteral": {
+        "doc": "Provides an array or object value, evaluating expressions in the elements of the array or values of the object.",
+        "example": {
+          "syntax": {
+            "method": ["JSON object or array"],
+            "result": "array | object"
+          },
+          "value": ["semiliteral",[["get", "label_x"], ["get", "label_y"]]]
+        },
+        "group": "Types",
+        "sdk-support": {
+          "basic functionality": {
+            "js": "https://github.com/maplibre/maplibre-style-spec/issues/950",
```

Comment:
The idea is to link to issues in the MapLibre GL JS and MapLibre Native repo so that people can track progress on the implementation.